lib/v2/nju/exchangesys.js

```diff
@@ -0,0 +1,67 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const dayjs = require('dayjs');
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type;
+    const type_dict = {
+        news: ['http://elite.nju.edu.cn/exchangesystem/index/moreList', 'http://elite.nju.edu.cn/exchangesystem/index/more?type=xw', '新闻通知'],
+        proj: ['http://elite.nju.edu.cn/exchangesystem/index/moreXmList', 'http://elite.nju.edu.cn/exchangesystem/index/more?type=xm', '交换生项目'],
+    };
+
+    // First get the `me` parameter
+    let meParam = null;
+    try {
+        const meResponse = await got({
+            method: 'get',
+            url: type_dict[type][1],
+        });
+        const me$ = cheerio.load(meResponse.data);
+        meParam = me$('title').attr('data-m');
+    } catch (e) {
+        // pass
+    }
+
+    let url = type_dict[type][0] + '?page=1&limit=20';
+    if (meParam) {
```

Comment:
The value of `.me` is simply the base64 encoded value of `sys.index` unless you're logged  in

Reply:
so we can simply hardcode the `.me` parameter?